where_to_go/lib/main.dart

```diff
@@ -7,119 +10,176 @@ void main() {
+    ),
+    Place(
+      titleText: "The longest city in Europe",
+      image: Assets.images.kr.provider(),
+      placeName: const Center(child: Text("Kryvyi Rih", style: TextStyle(fontSize: 24, fontWeight: FontWeight.bold))),
+      catchPhrase: const Center(child: Text("A life-long city")),
+      feature1: const Column(children: [Icon(Icons.iron), Text("Metals!")]),
+      feature2: const Column(children: [Icon(Icons.pets), Text("Green dogs")]),
+      feature3: const Column(children: [Icon(Icons.do_not_disturb), Text("Less criminal then before")]),
+    ),
+    Place(
+      titleText: "Big snowy mountains",
+      image: Assets.images.zakopane.provider(),
+      placeName: const Center(child: Text("Zakopane", style: TextStyle(fontSize: 24, fontWeight: FontWeight.bold))),
+      catchPhrase: const Center(child: Text("Zakopane - najbliżej Tatr")),
+      feature1: const Column(children: [Icon(Icons.snowboarding), Text("Snow sports!")]),
+      feature2: const Column(children: [Icon(Icons.hiking), Text("Cool routes to go hiking")]),
+      feature3: const Column(children: [Icon(Icons.water), Text("Oko morskie!")]),
+    ),
+    Place(
+      titleText: "GAMBLING",
+      image: Assets.images.gambling.provider(),
+      placeName: const Center(child: Text("Las Vegas", style: TextStyle(fontSize: 24, fontWeight: FontWeight.bold))),
+      catchPhrase: const Center(child: Text("LET`S GO GAMBLING")),
+      feature1: const Column(children: [Icon(Icons.money_off), Text("GAMBLING")]),
+      feature2: const Column(children: [Icon(Icons.attach_money), Text("GAMBLING")]),
+      feature3: const Column(children: [Icon(Icons.monetization_on), Text("GAMBLING")]),
+    ),
+    Place(
+      titleText: "I need to get there",
+      image: Assets.images.pentagon.provider(),
+      placeName: const Center(child: Text("Pentagon", style: TextStyle(fontSize: 24, fontWeight: FontWeight.bold))),
+      catchPhrase: const Center(child: Text("You know they built it in 1943?")),
+      feature1: const Column(children: [Icon(Icons.house), Text("5 sides")]),
+      feature2: const Column(children: [Icon(Icons.build), Text("Looks cozy")]),
+      feature3: const Column(
+          children: [Icon(Icons.build_circle), Text("5 aboveground floors,\nand 2 underground,\nofficially")]),
+    ),
+  ];
 
-  final String title;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text("Dream places"),
+      ),
+      body: ListView.builder(
+          itemCount: places.length,
+          itemBuilder: (context, index) {
+            final place = places[index];
+            return InkWell(
+                onTap: () async {
+                  // ignore: discarded_futures
+                  await Navigator.push<void>(
+                      context, MaterialPageRoute<void>(builder: (_) => DreamPlaceScreen(place: place)));
+                },
```

Comment:
```suggestion
                  await Navigator.push<void>(
                      context, MaterialPageRoute<void>(builder: (context) => DreamPlaceScreen(place: place)));
                },
```
ogólnie spoko praktyka to co zrobiłeś, ale z contexem rczej się tak nie robi bo można doprowadzić do dziwnych błędów przez przypadek. context się raczej nazywa context i tyle kropka / zawsze